src/controllers/resources/list/send-to-app.js

```diff
@@ -0,0 +1,59 @@
+export default (navClick, resourceBlock, selectedExportData, selectedImportData, objectPropExport, objectPropImport) => {
+  const { app, i18n, callApi } = window
+  const tabId = window.tabId
+  const Tab = window.Tabs[tabId]
+  navClick.click(() => {
+    const selectedResource = selectedExportData
+    if (!selectedResource.length > 0) {
+      app.toast(i18n({
+        en_us: 'Nothing was selected',
+        pt_br: 'Nada foi selecionado'
+      }))
+    } else {
+      navClick.find('button').attr('data-toggle', 'dropdown')
+      navClick.find('.dropdown-menu a').click((event) => {
+        const action = event.currentTarget && event.currentTarget.dataset && event.currentTarget.dataset.action
+        const appId = event.currentTarget && event.currentTarget.dataset && event.currentTarget.dataset.appId
+        const appName = event.currentTarget && event.currentTarget.dataset && event.currentTarget.dataset.id
```

Comment:
```suggestion
        const btnDataset = event.currentTarget && event.currentTarget.dataset
        if (!btnDataset) {
          return
        }
        const action = btnDataset.action
        const appId = btnDataset.appId
        const appName = btnDataset.id
```

além de evitar repetir `event.currentTarget && event.currentTarget.dataset && event.currentTarget.dataset` todas as linhas (😛), você não deveria prosseguir (inutilmente) se os valores não forem definidos (situação inesperada), certo?

Reply:
ok